tendrl/alerting/objects/definition/alerting.py

```diff
@@ -2,12 +2,46 @@
 data = """---
 namespace.tendrl.alerting:
   objects:
+    Definition:
+      enabled: True
+      help: "Definition"
+      value: _tendrl/definitions/alerting
+      list: _tendrl/definitions/alerting
+      attrs:
+        master:
+          help: Alerting definitions
+          type: String
     Config:
-      value: '/_tendrl/config/alerting'
-      data:
-        help: "The configurations path"
-        type: json
+      attrs:
+        data:
+          help: Configuration data of alerting for this Tendrl deployment
+          type: str
+        etcd_connection:
+          help: Host/IP of the etcd central store for this Tendrl deployment
+          type: str
+        etcd_port:
+          help: Port of the etcd central store for this Tendrl deployment
+          type: str
+        file_path:
+          default: /etc/tendrl/alerting/alerting.conf.yaml
```

Comment:
Why we need to maintain this path inside a file present at `/etc/tendrl/alerting/alerting.conf.yaml`? What I mean is, the config would be loaded from said path so it would be already hardcoded somewhere in code. Why we need to maintain the same value here if its already hardcoded.